packages/patternfly-4/react-docs/gatsby-browser.js

```diff
@@ -16,9 +16,9 @@ import '@patternfly/patternfly/patternfly.css';
 // Utilities
 import '@patternfly/patternfly/patternfly-addons.css';
 
-// Experimental components
+// Beta components
```

Comment:
Are these just temporarily necessary until a core change is made to include css in the main package?

Reply:
Just waiting on these last few [changes from core](https://github.com/patternfly/patternfly-next/commit/f24376e58d941420c477e0b0aa29f985af5111ae) to come in so we can delete the rest, until they come through we still need to manually import a couple of these